apis/core/index.ts

```diff
@@ -50,6 +51,12 @@ async function main() {
       password: env.maybe.STORE_ENDPOINTS_PASSWORD,
     },
     errorMappers,
+    middleware: {
+      operations: [
+        preferHydraCollection,
```

Comment:
This one is to select the collection operation in case when resource is `<> a hydra:Collection , hydra:Resource`. Causes ambiguity with the updated hydra-box/labyrinth